examples/music-bot/src/bot.ts

```diff
@@ -10,19 +10,17 @@ import { Track } from './music/track';
 import { MusicSubscription } from './music/subscription';
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
-const { token } = require('../auth.json');
+const { token, deployment_guild_id } = require('../config.json');
 
-const client = new Discord.Client({ intents: ['GUILD_VOICE_STATES', 'GUILD_MESSAGES', 'GUILDS'] });
+const client = new Client({ intents: ['GUILD_VOICE_STATES', 'GUILD_MESSAGES', 'GUILDS'] });
```

Comment:
```suggestion
const client = new Client({ intents: ['GUILD_VOICE_STATES', 'GUILDS'] });
```
`GUILD_MESSAGES` intent isn't needed